x/poolmanager/taker_fee_test.go

```diff
@@ -32,6 +34,7 @@ func (s *KeeperTestSuite) TestChargeTakerFee() {
 
 		expectedResult sdk.Coin
 		expectError    error
+		sendCoins      func(s *KeeperTestSuite, coin sdk.Coins)
```

Comment:
Would it be possible to make this a simple boolean field and then move the 

```
				s.App.BankKeeper.SendCoins(s.Ctx, s.TestAccs[nonWhitelistedSenderIndex], s.TestAccs[whitelistedSenderIndex], coins)

```

logic inside the actual test logic?